contracts/spec_interfaces/IDelegation.sol

```diff
@@ -15,17 +15,40 @@ interface IDelegations /* is IStakeChangeNotifier */ {
      * External functions
      */
 
-	/// @dev Stake delegation
+	/// Delegate your stake
+	/// @dev updates the election contract on the changes in the delegated stake
+	/// @dev updates the rewards contract on the upcoming change in the delegator's delegation state
+	/// @param to is the address to delegate to
 	function delegate(address to) external /* onlyWhenActive */;
 
+	/// Refresh the address stake for delegation power based on the staking contract
+	/// @dev Disabled stake change update notifications from the staking contract may create mismatches
+	/// @dev refreshStake re-syncs the stake data with the staking contract
+	/// @param addr is the address to refresh its stake
 	function refreshStake(address addr) external /* onlyWhenActive */;
 
+	/// Returns the delegated stake of an addr 
+	/// @dev an address that is not self delegating has a 0 delegated stake
+	/// @param addr is the address to query
+	/// @return delegatedStake is the address delegated stake
 	function getDelegatedStake(address addr) external view returns (uint256);
 
+	/// Returns the delegated stake of an addr 
```

Comment:
unclear
it returns the delegate address of the given delegator

Reply:
done